tsconfig.json

```diff
@@ -14,7 +14,13 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "strictNullChecks": true
```

Comment:
Shouldnt be needed since already using `strict: true` further above

See https://www.typescriptlang.org/tsconfig#strict